pylovo/config_data.py

```diff
@@ -106,12 +121,16 @@
     s_max bigint,
```

Comment:
Can you please change this "s_max" column and the respective code section to "transformer_size_selected"
```
            INSERT INTO building_clusters (version_id, plz, kcid, bcid, ont_vertice_id, **s_max**)
            VALUES (%(v)s, %(pc)s, %(k)s, %(count)s, %(t)s, %(l)s);
```